aten/src/ATen/core/dispatch/backend_fallback_test.cpp

```diff
@@ -79,6 +81,22 @@ void generic_wrapper_fallback(const c10::OperatorHandle& op, torch::jit::Stack*
   }
 }
 
+TEST(BackendFallbackTest, ConjugateTest) {
+  auto m = MAKE_TORCH_LIBRARY_IMPL(_, TESTING_ONLY_GenericMode);
+  m.fallback(torch::CppFunction::makeFromBoxedFunction<&conjugateFallback>());
+
+  c10::impl::IncludeDispatchKeyGuard guard(DispatchKey::Conjugate);
```

Comment:
You don't need this. Conjugate key is on the tensor, so that's how you will get to the conjugate implementation. Check the dispatcher slides again about this :)